presto-mongodb/src/test/java/io/prestosql/plugin/mongodb/TestMongoIntegrationSmokeTest.java

```diff
@@ -340,6 +340,18 @@ public void testDropTable()
         assertQueryFails("SELECT * FROM test.drop_table", ".*Table 'mongodb.test.drop_table' does not exist");
     }
 
+    @Test
+    public void testNullPredicates()
+    {
+        MongoCollection<Document> collection = client.getDatabase("testnull").getCollection("null_predicates");
+        collection.insertOne(new Document(ImmutableMap.of("Name", "abc", "Value", 1)));
+        collection.insertOne(new Document(ImmutableMap.of("Name", "abcd")));
+        collection.insertOne(new Document(Document.parse("{\"Name\": \"abcde\", \"Value\": null}")));
+        assertQuery("SELECT count(*) FROM testnull.null_predicates WHERE Value IS NULL OR rand() = 42", "SELECT 2");
```

Comment:
The generated `_schema` collection may miss the `Value` column if the document that doesn't have `Value` field was read first. It will lead to flaky tests. Let's define table definition explicitly. e.g.

```java
    @Test
    public void testNullPredicates()
    {
        assertUpdate("CREATE TABLE test.null_predicates(name varchar, value integer)");

        MongoCollection<Document> collection = client.getDatabase("test").getCollection("null_predicates");
        collection.insertOne(new Document(ImmutableMap.of("name", "abc", "value", 1)));
        collection.insertOne(new Document(ImmutableMap.of("name", "abcd")));
        collection.insertOne(new Document(Document.parse("{\"name\": \"abcde\", \"value\": null}")));

        assertQuery("SELECT count(*) FROM test.null_predicates WHERE value IS NULL OR rand() = 42", "SELECT 2");
        assertQuery("SELECT count(*) FROM test.null_predicates WHERE value IS NULL", "SELECT 2");
        assertQuery("SELECT count(*) FROM test.null_predicates WHERE value IS NOT NULL", "SELECT 1");

        assertUpdate("DROP TABLE test.null_predicates");
    }
```